feat: use cases - Prometheus (#90)
cncf/sig-app-delivery: operator-wg/whitepaper/07_Use_Cases.md

When updating an application, the operator contains the logic which is needed to get to the new application version and how to transition. As described in the last chapter, this could be mechanisms to backup data before updating and updating the database schema. Therefore, the logic included in the operator knows which prerequisites are necessary to build a consistent backup, how to backup the data and how to get back to the normal state.

Finally, the operator is able to remove the application and the resulting objects.

### Prometheus Operator

The Prometheus Operator was one of the first ever Operators written, along with etcd, that proved the use case for this problem space.

_"The Prometheus Operator serves to make running Prometheus on top of Kubernetes as easy as possible, while preserving Kubernetes-native configuration options."_

> When you install the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md), besides the operator controller pod/deployment, you have a wide range of APIs available to configure your own Prometheus stack. The APIs are represented as Custom Resource Definitions (CRDs) and those allow you to configure objects responsible, among other tasks, for:

Reviewer: ```suggestion
When the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md) is installed besides the operator controller pod/deployment, a wide range of APIs gets available to configure a Prometheus stack. The APIs are represented as Custom Resource Definitions (CRDs) which allow to configure objects responsible, among other tasks, for:
```
